packages/repo/src/mst/util.ts

```diff
@@ -120,3 +113,33 @@ export const cidForEntries = async (entries: NodeEntry[]): Promise<CID> => {
   const data = serializeNodeData(entries)
   return cidForCbor(data)
 }
+
+export const isValidMstKey = (str: string): boolean => {
+  const split = str.split('/')
+  return (
+    str.length <= 256 &&
```

Comment:
I *think* that this length check works ok. It is technically the number of bytes that should be <= 256, and a typescript string is UTF-16 and could be a different number of bytes. But I can't think of any evil strings off the top of my head where this code, with the `split('/')` and then double regex check, would ever have the "string" length be different from the "bytes" length.